### PR TITLE
Rewritten the addImdbShows.mako page, added show options UI functionality.

### PR DESCRIPTION
**Pull request by p0psicles**
... This should become the basis for other addXxxxShows pages.
- Copied the .enabled js function to comming/init, because i'd like t…o use it to toggle the choose show options UI for adding new shows.
- Rewritten the Choose rootDir/qualityOptions selection when adding a show through an IMDB list. The "add from trakt list" page, should use this interface. That's on my todo.
- Rewritten the addShowByID() function in webserve.py, to add passing the quality options.
- Made small change to debug log when retreiving a watchlist.
- Updated core.min.js because of changes to core.js

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [ ] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: gui/slick/js/core.js [hunk 1]
```diff
@@ -177,6 +177,19 @@ var SICKRAGE = {
                     $(this).prop('checked', $(bulkCheck).prop('checked'));
                 });
             });
+            
+            $(".enabler").each(function(){
```
Comment (p0psicles) [inline gui/slick/js/core.js:181]: @OmgImAlexis copied this to common/init, becuase i'd like to enabler functionality here. Ofcourse can also be moved to a more specific namespace?!

Comment (OmgImAlexis) [inline gui/slick/js/core.js:181]: I'll merge this into the branch and then have a look over. Is this branch finished? Should we start reviewing for it to be merged into dev?

**Comment by OmgImAlexis** [comment]
One thing I noticed is you keep putting a `*` at the start of your commits and none of them are actually linked to your github account. Can you please fix that before we look at merging into dev.

**Comment by labrys** [comment]
@p0psicles if you want to continue using markdown in your commit messages, that's fine, just keep in mind the first line is the title and doesn't accept markdown.

**Comment by p0psicles** [comment]
@OmgImAlexis I've linked the email that i've used in my github profile. I do think it only applies now to new commits. But why do you need to wait on that before merging? Not that I'm in a rush to move to dev. Just curious. I was in the assumption it's only used for the stats.

I just use the \* for better readability.

@labrys I'll make sure not to use the markdown in the title line.

**Comment by OmgImAlexis** [comment]
@p0psicles the main reason I ask is just like the code we try and keep everything uniform. Knowing exactly who committed, keeping clean commit messages, etc. is all part of that.

**Comment by p0psicles** [comment]
Okay great. Should be all good now.
On a sidenote. I don't know what you regards as a clean commit message. But sometimes I see commit messages in SR that are not informative at all. I'm willing to adjust were needed, but I expect nothing less from others.

@OmgImAlexis: I think you can start reviewing the imdb-watchlist branch any time now. I've been adding adding/moving code in core.js. Some of controller/actions can be named different. Would be nice if I   got some feedback on those.

Please don't merge into dev yet. Still need to rewrite the add from trakt page. And do some regression tests.

**Pull request merged**